### ISSUE TITLE
Hard to type Keywords

### ISSUE THREAD

**Issue by Kunniii** [dinhanhx/gir#7]
I can see some keywords such as `KếtQuả` `KhoáGiáTrị` or `KhoáGiáTrị`, etc. are hard to type in Vietnamese. My suggestion is to remove the dấu in the chữ. 
Back in the days without Unikey or Vietkey, we are used to Vietnamese with no dấu. Even though the old days have gone, the tập tính still remains so removing the dấu is not a big deal, right?
Does anyone face the same issue as mine?

**Comment by phunlh2001** [comment]
Yes, I have the same problem with you. I can't type the `KếtQuả` keyword. It's really difficult. I used Unikey for my Vietnamese typing, if I type the `KếtQuả` keyword I will get the `KeestQuar` after that

**Comment by phunlh2001** [comment]
> I can see some keywords such as `KếtQuả` `KhoáGiáTrị` or `KhoáGiáTrị`, etc. are hard to type in Vietnamese. My suggestion is to remove the dấu in the chữ. Back in the days without Unikey or Vietkey, we are used to Vietnamese with no dấu. Even though the old days have gone, the tập tính still remains so removing the dấu is not a big deal, right? Does anyone face the same issue as mine?

Yes, I have the same problem with you. I can't type the `KếtQuả` keyword. It's really difficult. I used Unikey for my Vietnamese typing, if I type the `KếtQuả` keyword I will get the `KeestQuar` after that

**Comment by Kunniii** [comment]
True, for me, the word `KếtQuả` for example, after I typed it would become `Ket61Qua3`.
Please update!!!

**Comment by nvhai245** [comment]
How about using underscore, like `Kết_Quả`? I still think keeping dấu is better.
